### ISSUE TITLE
helpp

### ISSUE THREAD

**Issue by attiqa680** [BMIRDS/3dMRISegmentation#4]
Hi, Hope you will be fine. Iam Research student from Comsats University Islamabad Pakistan. I have read your paper this is very remarkable paper and now iam trying to reproduce this paper for my research
There is issue in your code. When i run train.py it generate separate file for every epoch and there is 1200 epochs which generate 1200 epoch files. so please tell me in testing i have to give path for every single file which is very time consuming . Kindly help me
![Capture](https://user-images.githubusercontent.com/59041075/109763686-42fd0380-7c14-11eb-801e-7235d8c1d8a1.PNG)

**Comment by ntomita** [comment]
The training script saves only models that performs the best so far on the validation set. I'd suggest to use the most recent snapshot for testing, which is net-epoch-009 in your image.

**Comment by attiqa680** [comment]
Thankyou sir.
One more question I want to ask that at training there is 1200 epochs and
it will saves only best models but how we can find out which epoch is best.
Secondly I want this  results shown in figure.
